src/plugins/vis_type_table/public/components/table_vis_component_group.tsx

```diff
@@ -21,15 +21,17 @@ export const TableVisComponentGroup = memo(
   ({ tableGroups, visConfig, event, uiState }: TableVisGroupComponentProps) => {
     return (
       <>
-        {tableGroups.map(({ tables, title }) => (
+        {tableGroups.map(({ table, title }) => (
           <div key={title} className="visTable__group">
-            <TableVisComponent
-              title={title}
-              table={tables[0]}
-              visConfig={visConfig}
-              event={event}
-              uiState={uiState}
-            />
+            {table ? (
+              <TableVisComponent
+                title={title}
+                table={table}
+                visConfig={visConfig}
+                event={event}
+                uiState={uiState}
+              />
+            ) : null}
           </div>
```

Comment:
Does this mean is `table` is falsey, it would just add `<div ...></div>`? Is that what you wanted?

Reply:
There should always be table. If not we want to add <div ...></div> to avoid crash.

Reply:
A unit test here would help clarify the intent.